packages/xchain-dash/README.md

```
@@ -0,0 +1,108 @@
+- `clientKeystore` - Keystore-based client (`Client` default export alias)
+- `clientLedger` - Ledger hardware wallet client
+
+## Installation
+
+```
+yarn add @xchainjs/xchain-dash
+```
+
+Following peer dependencies have to be installed into your project. These are not included in `@xchainjs/xchain-dash`.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)